tox.ini

```diff
@@ -17,7 +17,7 @@ ignore = F821  ; TODO remove me when all examples in docstrings will be executab
 exclude=.venv, .git, .tox, dist, doc, build, gensim/models/deprecated
 
 [pytest]
-addopts = -rfxEXs --durations=20 --showlocals --reruns 3 --reruns-delay 1
```

Comment:
Yes, I will roll this back.

I disabled this because the tests I'm running for this branch are deterministic and it doesn't make sense to rerun them.